### ISSUE TITLE
clinical data

### ISSUE THREAD

**Issue by rongyua** [Cassie07/PathOmics#7]
It seems that you have made adjustments to the clinical data. Can you tell me what the adjustments are?

**Comment by Cassie07** [comment]
Hello @rongyua! Thanks for your interest in our study. I did not do any modification of the original clinical data. Any data preprocessing has been reflected in the code other than the original files. 

If the modification you mentioned is inferred by the name of the clinical data file (i.e., data_clinical_patient_modified.csv), I would like to clarify that it doesn't mean any content modification. It's just because I converted the original TXT file to a CSV file. To make it clear to myself, I added "modified" in its name. 

You can directly use the clinical data downloaded from CbioPortal. Also, please let me know if you have any other questions about the clinical data.

**Comment by rongyua** [comment]
During the fine-tuning phase, the test C-index seems to be generally higher than the training and validation C-index. Is this normal?

**Comment by Cassie07** [comment]
@rongyua  It might happen. Also, you can use different random seeds for dataset split and report the average C-index among random seeds as a more convincing and robust result.

**Comment by rongyua** [comment]
During fine-tuning, when the valC index is high, test is very low, and when val is very low, test is very high.

**Comment by rongyua** [comment]
There is a problem that the test loss is smaller than the training loss during both training and testing.

**Comment by Cassie07** [comment]
Hello @rongyua, thanks for your interest in our study! I think the test loss is smaller than the training loss, which can be possible for several reasons. I listed two common reasons for reference; you are welcome to provide more thoughts:
(1) The dropout is used in the code. Generally, the dropout is activated when training but deactivated when evaluating the validation set, which means the task in the training is more difficult than in the validation.
(2) Some difficult cases are in the training set but not in the validation set, i.e., the training set is unrepresentative of the validation set. Because of the random dataset split, this case can happen occasionally.

Also, based on your previous comments, do you have any suggestions or anything else you think I can do to help? Could you please provide me with your questions or demands directly so I can better assist you?

**Another thing I am wondering is if it is difficult for you to say a simple ‘hello’ or ‘thank you’ at least once in our discussion. I would really appreciate it if you could respect the contributors in the GitHub community. We are unpaid for answering questions or releasing the code, and we do not owe anyone.**

Still, good luck with your research, and thanks for your interest in our study! I am still glad to answer your further questions if you still need help figuring out anything.

**Comment by rongyua** [comment]
Thank you very much for your prompt reply and your contribution to the paper, which has benefited me a lot, and I also express my apology for the harm I have caused you by not paying attention to your tone when seeking answers to your questions. I'm very sorry. This is a very innovative paper and it has made a great contribution to my research. However, I have encountered some problems that I have not encountered in previous experiments. I am eager to know the answers so that it will cause trouble to you. I once again express my deep apologies for the inconvenience and wish you success in your scientific research! ! !

**Comment by Cassie07** [comment]
Hi @rongyua, thanks for your understanding and kindness! Let's first sync both of our thoughts to see whether I can provide more help to you. Please let me know your answer to the following question:

1. Have you tried to modify any part of the architecture, training process, or even dataset? If so, please feel free to let me know if you need any modifications.
2. As I mentioned above, the test loss could be smaller than the training loss in several situations. Is this a crucial issue that affects your experiments or results? I didn't get the point of why the test loss must be smaller than the training loss. Do you have further concerns about this observation? Will it hurt your model performance or something else?

I will also rerun the code to see whether I can help you resolve the concerns. Please feel free to let me know your thoughts.

**Comment by Cassie07** [comment]
Hi @rongyua, I just reran the code of the experiments on the baseline model and our proposed method. So far, I find that:

1. The test loss can be smaller than the training loss in some epochs in both the baseline and our method. 
2. In the finetuning stage, the test C-Index can be higher than the validation and training C-Index, but not by a large margin, as you mentioned,  and vice versa.

So, I assume that the observation you mentioned above could happen but not affect the model performance. Please feel free to let me know your thoughts.

**Comment by rongyua** [comment]
Hello,@Cassie07 ，Thank you again for your timely reply. When I repeated the experiment, the two situations mentioned before occurred at a high rate. I suspected that there was something wrong with my downloaded data. Please provide the clinical patient file data_clinical_patient_modified.csv for your convenience. and miRNA files, or provide the address where the files can be downloaded? I would be very grateful if you could.

**Comment by rongyua** [comment]
Hello， @Cassie07 .Sorry, I only saw your latest reply before. My current idea is to improve based on your paper, but the above problems occurred during pre-training and fine-tuning. I did not make any changes to the code. I want the code results It is in a normal state for subsequent improvements. It may be a problem with some hyperparameters. Thank you very much for your timely reply.
